src/java/org/apache/cassandra/metrics/CacheMetrics.java

```diff
@@ -93,42 +93,7 @@ public void reset()
     {
         hits.mark(-hits.getCount());
         misses.mark(-misses.getCount());
```

Comment:
I just want to point out that this approach is not performing an actual `reset`. It will put the `Meter` counter to zero but will not reset the moving averages but rather inject a weird value into them.
This method being only used by `CacheMetricsTest` and `CachingBench` , I do not think that it makes sense to attempt to fix the problem but it might be good to let a comment that there is a known issue here.